src/Subscriber/RewriteUnwantedPhrasesSubscriber.php

```diff
@@ -59,7 +59,10 @@ public function onPullRequest(GitHubEvent $event): void
         ]);
     }
 
-    private function replaceUnwantedPhrases(string $text, &$count)
+    /**
+     * @param int &$count (optional)
```

Comment:
should be `@param int|null &$count` with `@param-out int`, to document that it is OK to have a `null` value when calling the method, but that it won't ever be `null` after the call.

Reply:
and this could even use `int<0, max>|null` as it looks like Psalm wants a more precise type.

Reply:
Thanks!